### ISSUE TITLE
Estimate Power and resulting Normalised Power

### ISSUE THREAD

**Issue by RalphPorter** [GoldenCheetah/GoldenCheetah#3879]
Hi,

I've imported my rides from Garmin Connect to add power estimates and I've noticed that the normalised Power is often well overstated, the average power looks sensible but for example on one ride the avg watts was 157 and the normalised was 262, this is massively overstated I'd expect maybe 165 to 170.  I can provide the TCX file if it helps.

Many Thanks,

Ralph.

Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers

**Comment by RalphPorter** [comment]
Hi,

This is a bug, I've been writing software for over 30 years and certainly in my experience when said software produces incorrect results it generally has a bug, admittedly Garmin could possibly be at fault if their TCX data is wrong but I highly doubt that so my money is very much on a bug in the Golden Cheetah software. 

Totally unhelpful response from the moderator.

**Comment by amtriathlon** [comment]
We have 2 forums for that kind of discussions, use whatever you find more convenient.
